sphinx/api/rosie-web.rst

```diff
@@ -152,7 +152,7 @@ REST API
 
       .. code-block:: http
 
-         GET http://host/my_prefix/search?var+bob+nowcast&format=json HTTP/1.1
+         GET http://host/my_prefix/search/var+bob+nowcast&format=json HTTP/1.1
```

Comment:
Why doesn't the old syntax work any more?

Reply:
I don't think this syntax was ever correct, was it?

Reply:
That's true.

Reply:
It's probably either an error copied forward from the old docs or a translation error by yours truely :frowning:

Reply:
Isn't this meant to instead be:
```suggestion
         GET http://host/my_prefix/search?s=var+bob+nowcast&format=json HTTP/1.1
```

The original, as with your change suggestion, both don't make sense as URL queries, but this does? To check I am not completely mistaken, I tried these URLs out on both the current ("old") & my new (PR #2288) versions of the rosie discovery utility, & the original & change here both give a 404 error.